src/pages/questions/QuestionDisplay.jsx

```diff
@@ -57,16 +60,55 @@ const getValueTagExamplesURL = (question) => {
   return "";
 };
 
-const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
+const getNbOfQuestionForValue = async ({ type, valueTag }) => {
+  const { data: dataFetched } = await robotoff.questions(
+    "random",
+    type,
+    valueTag,
+    null,
+    null,
+    1
+  );
+  return dataFetched.count;
+};
+
+const QuestionDisplay = ({
+  question,
+  answerQuestion,
+  resetFilters,
+  filterState,
+}) => {
   const { t } = useTranslation();
   const valueTagQuestionsURL = getValueTagQuestionsURL(question);
   const valueTagExamplesURL = getValueTagExamplesURL(question);
+  const [nbOfPotentialQuestion, setNbOfPotentialQuestions] =
+    React.useState(null);
 
   React.useEffect(() => {
+    if (filterState.valueTag) {
+      // value is already in the filter so it's a useless information
+      setNbOfPotentialQuestions(null);
+      return;
+    } else {
+      let validRequest = true;
+      getNbOfQuestionForValue({
+        type: question?.insight_type,
+        valueTag: question?.value_tag,
+      }).then((nbQuestions) => {
+        if (validRequest) {
+          setNbOfPotentialQuestions(nbQuestions);
+        }
+      });
```

Comment:
maybe add a catch() {}

Reply:
If a network error occurs, what should we do? I don't get the use of the `catch` here

Reply:
yeah, mmmm, the issue I'm thinking is that if an error accurs in the promise it will cause (from what I understand) the whole app to fail (unless there's  an error handler somewhere in the tree). I don't know what would work here, an option would be to have an error state and show it in the UI, but I guess that even just logging the error or doing nothing ... what do you think @alexfauquette ?